docker/generate_makefile.py

```diff
@@ -4,303 +4,163 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-# http://www.apache.org/licenses/LICENSE-2.0
+#      http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Simple generator for local Makefile rules."""
+"""Generates Makefile containing docker image targets."""
```

Comment:
Consider focusing on one thing in one PR. Perhaps do the GCB spec generation in this PR, and the Makefile in the next one (or the other way around)?

Reply:
Okay. I'll store the changes for cloud build for the next CL.